perceval/converters/cqasm_converter.py

```diff
@@ -226,38 +254,9 @@ def convert_file(
         as the last gate. Otherwise, uses only `heralded CNOT`
         :return: the converted processor
         """
-        ast = cqasm.Analyzer().analyze_file(source_file_name)
-        return self.convert(ast, use_postselection)
-
-    def convert_string_v1(
-            self,
-            source_string: str,
-            use_postselection: bool = True) -> Processor:
-        r"""Convert a cQASM v1 quantum program into a `Processor`.
-
-        :param source_string: The cQASM program stored in a string
-        :param use_postselection: when True, uses a `postprocessed CNOT`
-        as the last gate. Otherwise, uses only `heralded CNOT`
-        :return: the converted processor
-        """
-        lines = source_string.split('\n')
-        ast = self._v3_ast_from_v1_source(lines)
-        return self.convert(ast, use_postselection)
-
-    def convert_file_v1(
-            self,
-            source_file_name: str,
-            use_postselection: bool = False) -> Processor:
-        r"""Convert a cQASM v1 quantum program into a `Processor`.
-
-        :param source_file_name: The path to the cQASM program to load
-        :param use_postselection: when True, uses a `postprocessed CNOT`
-        as the last gate. Otherwise, uses only `heralded CNOT`
-        :return: the converted processor
-        """
-        lines = open(source_file_name, 'r').readlines()
-        ast = self._v3_ast_from_v1_source(lines)
-        return self.convert(ast, use_postselection)
+        return self.convert_string(
+            open(source_file_name, "r").read(),
+            use_postselection)
 
```

Comment:
Moved the import to the class level.

Removed the TODO. Waiting for @maxwell04-wq to make her changes on this part of the code and improve the v1 parser.

Reply:
@burlemarxiste besides the rotation gates, what other changes do you suggest? I'll add the support for rotation gates shortly.

Reply:
I agree with @burlemarxiste

Reply:
Hello @burlemarxiste and @maxwell04-wq ,

We have until Wednesday to merge and close the issue, so that UnitaryHACK takes it into account. From what I understand, we might want a few tests supporting qasm v1 format from @maxwell04-wq and I think that's all. Is that right?
@maxwell04-wq , do you think you can do this before Wednesday?

Thanks,
Eric

Reply:
Hi @raksharuia and @ericbrts, 

As today is the last day to claim the bounty, I'll be on standby for any changes needed. I have implemented the requested features, however, I still feel like a parser like `libqasm` would have been much neater. In any case, I hope my implementation serves the purpose.

@burlemarxiste has done an amazing job at this PR, and this shouldn't go to waste 😊